### PR TITLE
Check for everyone ping and add code plugin

### LINKED ISSUE #481: Check for everyone ping in command handler

### PR DESCRIPTION
**Pull request by tanndlin**
Fixes #481

### CODE REVIEW

**Comment by tanndlin** [comment]
Can bring back code plugin after this
Though, I still feel like it needs to be revamped or something

**Comment by suneettipirneni** [comment]
This may work for other plugins but it won't work for the code plugin, because it references another message

**Comment by tanndlin** [comment]
> This may work for other plugins but it won't work for the code plugin, because it references another message

You right

**Comment by suneettipirneni** [comment]
> I dont think this works bc it doesnt show up as @everyone.
> It's like a normal mention `<@4085743750>`
> 
> You can just do message.mentions?.everyone

I tested it, and it prevented the code plugin from repeating `@everyone`. The reason is because the code formatting converts everything into plain text, so transforming mentions like, `<@4085743750>` isn't a thing that discord does.

File: src/app/plugins/code.plugin.ts [hunk 1]
```diff
@@ -39,9 +39,11 @@ export class CodePlugin extends Plugin {
       .fetch(messageID)
       .then((targMessage) => {
 
+        console.log(targMessage.mentions);
+
         // Check if the target language has '@everyone' in it.
         if (targMessage.content.match(/@everyone/) != null) {
```
Comment (tanndlin) [inline src/app/plugins/code.plugin.ts:45]: `if(!targMessage.content.match(/@everyone/))`

File: src/app/plugins/code.plugin.ts [hunk 2]
```diff
@@ -39,9 +39,11 @@ export class CodePlugin extends Plugin {
       .fetch(messageID)
       .then((targMessage) => {
 
+        console.log(targMessage.mentions);
```
Comment (tanndlin) [inline src/app/plugins/code.plugin.ts:42]: left over log

**Review by suneettipirneni — APPROVED** [review]

**Pull request merged**